src/components/tab-controls/tab-controls.scss

```diff
@@ -33,21 +37,19 @@
 		border-color: currentColor;
 	}
 
-	:checked + &:hover {
-		border-color: $color-focus-state;
-	}
-
-	.tab-controls__input:focus + & {
```

Comment:
please add a focus state back.  i know it wasn't a GOOD focus state, but it was better than nothing

Reply:
OK! I talked it over with Julie, there is a place on the Marble queue now to review focus states https://www.figma.com/file/2xBciVFQGyisAVWHNTxP8T/Marble-Design-System-and-Component-Library?node-id=281%3A0